### PR TITLE
Add new advanced desktop targeting for new users with infrequent use

### PR DESCRIPTION
**Pull request by lmegviar**
This new targeting will support out follow up import bookmarks experiments. The goal is to target users who used Firefox 6 or fewer times over the past 28 days and who have a profile age of less than 28 days.

### CODE REVIEW

**Review by jaredlockhart — APPROVED** [review]
Looks good 👍

**Comment by lmegviar** [comment]
@aminomancer I'm unable to add additional reviewers, but if you have time to give this a second set of OMC eyes I'd appreciate it.

**Comment by lmegviar** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1806154

**Review by aminomancer — APPROVED** [review]
Looks good thanks!

File: app/experimenter/targeting/constants.py [hunk 1]
```diff
@@ -461,6 +461,20 @@ def __post_init__(self):
...
 )
 
+NEW_USERS_WITH_INFREQUENT_USE = NimbusTargetingConfig(
+    name="New users with infrequent use",
+    slug="new_users_with_infrequent_use",
+    description="0 - 6 days activity in past 28 days and profile age <= 28 days",
+    targeting=(
+        f"{PROFILE28DAYS} "
+        "&& userMonthlyActivity|length <= 6"
+    ),
```
Comment (aminomancer) [inline app/experimenter/targeting/constants.py:471]: Oh I got this formatting error in my previous PR. Try this?
```suggestion
    targeting=f"{PROFILE28DAYS} && userMonthlyActivity|length <= 6",
```

Comment (jaredlockhart) [reply app/experimenter/targeting/constants.py:471]: I can fix it don't worry about it

**Pull request merged**